test/helpers.js

```diff
@@ -73,8 +73,12 @@ export async function getTestUser() {
   return organization.getUser(userId)
 }
 
-export async function getDefaultSpace() {
+export async function getDefaultSpace(plainClient = false) {
   const { spaceId } = TestDefaults
+
+  if (plainClient) {
```

Comment:
Why was this change needed here?

Reply:
to create a different client base on the default space, though @Inbal-Gordon we can probably improve it:

- We can undo the change on this `helpers.js`
- In `entry-integration` `line 413` we can use ` plainClient = initPlainClient({ spaceId: TestDefaults.spaceId })` instead. Make sure the imports are working.